src/components/Calculator.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './calculator.css';
+
+const Calculator = () => (
+  <div className="calculator-container">
+    <div className="input">
+      <p className="values">0</p>
+    </div>
+
+    <div className="signs">
+      <button type="button" className="sign btn1">AC</button>
+      <button type="button" className="sign btn2">+/-</button>
+      <button type="button" className="sign btn3">%</button>
+      <button type="button" className="sign sing-math btn4">+</button>
+      <button type="button" className="sign btn5">7</button>
+      <button type="button" className="sign btn7">8</button>
+      <button type="button" className="sign btn8">9</button>
+      <button type="button" className="sign sing-math btn9">X</button>
+      <button type="button" className="sign btn10">4</button>
+      <button type="button" className="sign btn11">5</button>
+      <button type="button" className="sign btn12">6</button>
+      <button type="button" className="sign sing-math btn13">-</button>
+      <button type="button" className="sign btn14">1</button>
+      <button type="button" className="sign btn15">2</button>
+      <button type="button" className="sign btn16">3</button>
+      <button type="button" className="sign sing-math btn17">+</button>
+      <button type="button" className="sign btn18">0</button>
+      <button type="button" className="sign btn19">.</button>
+      <button type="button" className="sign sing-math btn20">=</button>
+    </div>
+  </div>
+);
+
+export default Calculator;
```

Comment:
- [ ] According to the project requirements, the calculator component should at least have 1 child component, kindly implement this
![image](https://user-images.githubusercontent.com/62705947/228265767-449bcec3-0f8b-4501-91e8-7232315dd3e2.png)